Documentation/devel/user-namespaces.md

```diff
@@ -4,45 +4,35 @@ User namespaces
 Background
```

Comment:
Headings in this file need to be ATX: `# Heading`.

`s/Background/## Background/`